Nokwanda

KwandoV opened the pull request.

KwandoV committed:
implementing the booking sys

KwandoV committed:
booking system done...i hope

phasha101 reviewed (changes requested):
app secrests are not to be pushed, use a folder and place it in .git.ignore

phasha101 committed:
Update skillsync-706d4-firebase-adminsdk-bkiyy-95f6231b51.json

secrets detected and removed

phasha101 committed:
Update auth.py

more secrets found and removed, you must create a folder called secret files and add all secrets as files in it then add the secrets to gitignore by typing secret files at the bottom if the file .gitignore
